helm/charts/kratos/templates/secrets.yaml

```diff
@@ -8,5 +8,6 @@ metadata:
 type: Opaque
 data:
   dsn: {{ include "kratos.dsn" . | b64enc | quote }}
-  # Generate a random secret if the user doesn't give one. User given password has priority
-  secretsSession: {{ include "kratos.secrets.cookie" . | required "Value kratos.config.secrets.cookie can not be empty!" | b64enc | quote }}
+  # Generate a random secret if the user doesn't give one. User given secret has priority
+  secretsDefault: {{ ( include "kratos.secrets.default" . | default ( randAlphaNum 32 )) | required "Value kratos.config.secrets.default can not be empty!" | b64enc | quote }}
```

Comment:
Did you check if `randAlphaNumeric` is a cryptographically secure PRNG?

Reply:
Honestly I just copied that from the hydra chart.